course/templates/course/_siblings.html

```diff
@@ -4,7 +4,11 @@
 <nav class="row siblings">
 
     {% if next %}
-    {% module_accessible next as accessible %}
+     {% if next.submittable %}
+      {% exercise_accessible next as accessible %}
```

Comment:
(I can't target this comment to the bottom of the file.) In the end of this file, there is the link to the "previous" item. This part is for the "next" item. Did you test the previous link? It could require the same separation of submittable and non-submittable items. Now, it always uses the `module_accessible` value.

Reply:
Added the same separation between exercises and modules.